.config/tsaoptions.json

```diff
@@ -1,8 +1,8 @@
 {
   "notificationAliases": ["chasun@microsoft.com"],
-  "areaPath": "Vienna\\ONNX Runtime\\Shared Core",
-  "codebaseName": "onnxruntime_master",
-  "instanceUrl": "https://msdata.visualstudio.com/",
-  "projectName": "Vienna",
+  "areaPath": "ONNX Runtime\\Engineering Systems",
+  "codebaseName": "onnxruntime_github",
+  "instanceUrl": "https://aiinfra.visualstudio.com/",
+  "projectName": "ONNX Runtime",
   "ignoreBranchName": true
-}
\ No newline at end of file
+}
```

Comment:
Please add 
```
"template": "AIINFRA_TSA"
```
to the json object. Usually we don't need to explicitly set it. But I don't know why this time it is not working.